types/react-slider/react-slider-tests.tsx

```diff
@@ -1,15 +1,18 @@
 import * as React from 'react';
 import ReactSlider from 'react-slider';
```

Comment:
Use the commonjs syntax `import ReactSlider = require('react-slider')`, and change `"esModuleInterop": false` in tsconfig.

Consumers of the package who want to use `import ReactSlider from 'react-slider'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.